distribution/ddf-common/src/main/resources-filtered/etc/system.properties

```diff
@@ -103,6 +103,22 @@ solr.data.dir=_DO_NOT_EXPAND_${karaf.home}/data/solr
 org.codice.ddf.system.threadPoolSize=128
 
 
+
+#
+# Intrigue Settings
```

Comment:
✏️ These should probably be added to https://github.com/codice/ddf/blob/17be0101150f311503a120f261ca709537ac4cee/distribution/docs/src/main/resources/content/_configuring/global-settings.adoc as well.